src/radical/pilot/raptor/worker_mpi.py

```diff
@@ -554,47 +612,68 @@ def _dispatch_function(self, task, env):
         func = task['description']['function']
 
         to_call = ''
+        names   = ''
         args    = task['description'].get('args',   [])
         kwargs  = task['description'].get('kwargs', {})
+        py_func = False
+
+        self._log.debug('=== orig args: %s : %s', args, kwargs)
 
         # check if we have a serialized object
+        self._log.debug('func serialized: %d: %s', len(func), func)
         try:
-
+            # FIXME: can we have a better test than try/except?  This hides
```

Comment:
I totally agree with this example: if we have an issue on the deserialization level, it's totally hidden, and won't know the type of error because we do not have logs from the sterilizer side. So it is an issue (I faced it recently).

Reply:
Could you let `get_func_attr` return a reasonable error for example?

Reply:
Anyway, I think tht is out of scope for this specific PR, so will merge now...